cmd/config/set.go

```diff
@@ -33,7 +33,7 @@ func getSetCmd(ks meta.IKubescape) *cobra.Command {
 }
 
 var supportConfigSet = map[string]func(*metav1.SetConfig, string){
-	"accessKey":      func(s *metav1.SetConfig, accessKey string) { s.AccessKey = accessKey },
+	"access-key":     func(s *metav1.SetConfig, accessKey string) { s.AccessKey = accessKey },
```

Comment:
Now access key is having a different case than other configs... why don't we align others as well?